src/components/pages/Home/Onboarding/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import styles from './index.module.css'
+import Header from './Header'
+import Main from './Main'
+import Navigation from './Navigation'
+import Container from '../../../atoms/Container'
+import Stepper from './Stepper'
+import DownloadMetamask from './Steps/DownloadMetamask'
+import ConnectAccount from './Steps/ConnectAccount'
+import ConnectNetwork from './Steps/ConnectNetwork'
+import ImportOceanToken from './Steps/ImportOceanToken'
+import Ready from './Steps/Ready'
+import ClaimTokens from './Steps/ClaimTokens'
+import { useUserPreferences } from '../../../../providers/UserPreferences'
+import { useWeb3 } from '../../../../providers/Web3'
+import { GX_NETWORK_ID } from '../../../../../chains.config'
+
+export interface OnboardingStep {
+  title: string
+  subtitle: string
+  body: string
+  image?: {
+    childImageSharp: {
+      original: {
+        src: string
+      }
+    }
+  }
+  buttonLabel?: string
+  buttonSuccess?: string
+}
+
+const steps = [
+  { shortLabel: 'MetaMask', component: <DownloadMetamask /> },
+  { shortLabel: 'Connect', component: <ConnectAccount /> },
+  { shortLabel: 'Network', component: <ConnectNetwork /> },
+  { shortLabel: 'Tokens', component: <ImportOceanToken /> },
+  { shortLabel: 'Faucet', component: <ClaimTokens /> },
+  { shortLabel: 'Ready', component: <Ready /> }
+]
+
+export enum NavigationDirections {
+  PREV = 'prev',
+  NEXT = 'next'
+}
+
+export default function OnboardingSection(): ReactElement {
+  const { accountId, balance, networkId, web3Provider } = useWeb3()
+  const { onboardingStep, setOnboardingStep } = useUserPreferences()
+  const [onboardingCompleted, setOnboardingCompleted] = useState(false)
+  const [navigationDirection, setNavigationDirection] =
+    useState<NavigationDirections>()
+  const stepLabels = steps?.map((step) => step.shortLabel)
```

Comment:
question(non-blocking): when could `steps` be empty or undefined?

Reply:
It's a leftover from the previous implementation when it was passed as a prop, my bad.